content/en/docs/tutorials/hello-minikube.md

```diff
@@ -236,22 +235,22 @@ Minikube has a set of built-in addons that can be enabled, disabled and opened i
     pod/storage-provisioner                     1/1       Running   0          34m
 
     NAME                           TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)             AGE
-    service/heapster               ClusterIP   10.96.241.45    <none>        80/TCP              26s
+    service/metrics-server               ClusterIP   10.96.241.45    <none>        80/TCP              26s
```

Comment:
Perhaps, we could improve alignment here? Or it isn't an issue in a rendered form?

Reply:
@php-code the [deploy preview](https://deploy-preview-18673--kubernetes-io-master-staging.netlify.com/docs/tutorials/hello-minikube/) shows how this will look (see the **deploy/netlify** section of the checks list for a link to the preview).

Seems like your concern is valid.
/lgtm cancel